README.md

```diff
@@ -16,3 +16,4 @@ decentralized and appless.
 ## Getting Started
 
 ## License
+Copyright 2017 - 2018 Lunchclass. and the Absolute project authors. Licensed under Apache License v2.0.
```

Comment:
I made a mistake in Issue #396, that period of the copyright should be 2017. @faruk13, would you modify it again?